lib/scripting/post-process-expr.spec.ts

```diff
@@ -46,9 +46,9 @@ describe('The VBScript transpiler - Math', () => {
 	});
 
 	it('should transpile a "Not" expression', () => {
-		const vbs = `EnableBallControl = Not 10\n`;
+		const vbs = `EnableBallControl = Not EnableBallControl\n`;
 		const js = vbsToJs(vbs);
-		expect(js).to.equal('EnableBallControl = ~10;');
+		expect(js).to.equal('EnableBallControl = !EnableBallControl;');

```

Comment:
Cool, I came across this bug while testing my state updates :)

Reply:
I think we still need to look into this, as `NOT` could be `~` when the literal is an integer. I believe I read this somewhere, but of course now I can't find it 😃

Reply:
Oh, I see. Some more context-aware logic then...